packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -1028,7 +1031,13 @@ class MongoshNodeRepl implements EvaluationListener {
    */
   runtimeState(): MongoshRuntimeState {
     if (this._runtimeState === null) {
-      throw new MongoshInternalError('Mongosh not initialized yet');
+      // This error can be really hard to debug, so we always attach stack traces
+      // from both when .close() was called and when
+      throw new MongoshInternalError(
+        `mongosh not initialized yet\nCurrentTrace: ${
+          new Error().stack
+        }\nClose trace: ${this.closeTrace}\n`
```

Comment:
is there a reason we're ending with a newline?

Reply:
Yeah no, good point. I think I have a tendency to always terminate string that contain multiple newlines with one (just like most editors do for files), but Errors are a bit special in that their messages are usually displayed with an extra newline anyway. Removed in 3bcdc5295025302154e06fa5f746f150aa5ca17e!